src/dashboard/components/navbar_component.py

```diff
@@ -143,22 +139,43 @@ def navbar_component() -> Component:
         ],
     )
 
-def generate_navbar_contents(page_registry: PageRegistry, item_to_highlight: Optional[str] = None
+
+def generate_navbar_contents(
+    page_registry: PageRegistry, item_to_highlight: Optional[str] = None
 ) -> list[html.Div]:
-    """Generate a list containing two divs. First div contains links to all 
-    pages, second div contains buttons and toggles.
+    """Generate a list containing two divs.
 
-    Read docstring for generate_navbar_page_items for first div. And #TODO second name
+    First div contains links to all pages, second div contains logout
```

Comment:
The second div will not only contain logout. In the future it will be filled with non-link buttons

Reply:
Added "will contain settings etc in future"